parsons/etl/etl.py

```diff
@@ -9,6 +9,37 @@ class ETL(object):
     def __init__(self):
         pass
 
+    def head(self, n=5):
+        """
+        Return the first n rows of the table
+
+        `Args:`
+            n: int
```

Comment:
Oh I guess there's also the question of whether `n` is the right variable name. Generally we try to avoid single letter variable names, but if we were going to make an exception this seems pretty reasonable as "n" is generally understood to mean "number". But it might be more readable to name this "rows".

Reply:
I believe n=5 is the standard for this function in most cases. That is how they do it in Pandas: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.head.html

Reply:
I'm fine with keeping it as is. If any single-letter variable is comprehensible, it's this one.